drivers/watchdog/Kconfig

```diff
@@ -8,6 +8,7 @@
 
 menuconfig WATCHDOG
 	bool "Watchdog Support"
+	select HAS_DTS_WDT
```

Comment:
Can't we simply get rid of it?

Reply:
We can, I wanted to keep it around until after the release.